src/compiler/index.ts

```diff
@@ -1,4 +1,4 @@
 import './polyfills';
 import {compile} from './compile';
 
-globalThis['compile'] = compile;
+(globalThis as any)['compile'] = compile;
```

Comment:
What is the type of `globalThis`? Can we just say it`s `any` since we literally always would need to cast it like this?

Reply:
Hmm. Actually. For `compiler` specifically it would be a good idea to type out our expectation on the global object

Reply:
This was less pretty than I thought it would be 😬